src/server/game/Entities/AreaTrigger/AreaTrigger.cpp

```diff
@@ -275,7 +275,18 @@ bool AreaTrigger::CreateServer(Map* map, AreaTriggerTemplate const* areaTriggerT
 
     auto areaTriggerData = m_values.ModifyValue(&AreaTrigger::m_areaTriggerData);
     SetUpdateFieldValue(areaTriggerData.ModifyValue(&UF::AreaTriggerData::BoundsRadius2D), GetMaxSearchRadius());
-    SetUpdateFieldValue(areaTriggerData.ModifyValue(&UF::AreaTriggerData::DecalPropertiesID), 24); // blue decal, for .debug areatrigger visibility
+    if (position.SpellForVisuals)
+    {
+        if (SpellInfo const* spellInfo = sSpellMgr->GetSpellInfo(position.SpellForVisuals, DIFFICULTY_NONE))
+        {
+            SetUpdateFieldValue(areaTriggerData.ModifyValue(&UF::AreaTriggerData::SpellForVisuals), position.SpellForVisuals);
+            SetUpdateFieldValue(areaTriggerData.ModifyValue(&UF::AreaTriggerData::SpellVisual).ModifyValue(&UF::SpellCastVisual::SpellXSpellVisualID), spellInfo->GetSpellVisual());
```

Comment:
This is wrong, GetSpellVisual() returns ID for SpellVisual.db2 but you need SpellXSpellVisual here